libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -218,7 +218,12 @@ bool GLSLPostProcessor::process(const std::string& inputShader,
     switch (mOptimization) {
         case MaterialBuilder::Optimization::NONE:
             if (mSpirvOutput) {
-                GlslangToSpv(*program.getIntermediate(mShLang), *mSpirvOutput);
+                SpvOptions options;
+#ifndef NDEBUG
+                options.generateDebugInfo = true;
```

Comment:
Are these options necessary for the `analyze-spirv` flag? If so, maybe we should only allow `analyze-spirv` in debug builds of `matinfo`.

Reply:
This is orthogonal to the matinfo changes.